src/GafferSceneUI/Inspector.cpp

```diff
@@ -291,14 +296,25 @@ void Inspector::inspectHistoryWalk( const GafferScene::SceneAlgo::History *histo
 						if( nonEditableReason.empty() )
 						{
 							result->m_editFunction = [source = result->m_source] ( bool unused ) { return source; };
+							result->m_disableEditFunction = disableEditFunction( result->m_source.get(), history );
 							result->m_editWarning = editWarning;
 						}
 						else
 						{
 							result->m_editFunction = nonEditableReason;
+							result->m_disableEditFunction = nonEditableReason;
 						}
-
-						result->m_disableEditFunction = disableEditFunction( result->m_source.get(), history );
+					}
+					else if( node->ancestor<EditScope>() && node->ancestor<EditScope>() != result->m_editScope )
+					{
+						result->m_disableEditFunction = fmt::format(
+							"Edit authored in EditScope ({}), set it as the target EditScope to disable.",
```

Comment:
This isn't unique to this PR, but I find messages of this form are often hard to read because my EditScope is often called EditScope, leading to a sentence full of EditScope. Maybe that's just because I'm throwing scenes together for testing, and everyone else has better names? But I wonder if we should refer to "the scope in which you have chosen to edit" as the "edit scope" rather than the "EditScope", to help with this?

I also wonder if there could be clarity to be gained by relegating the changeable details to a second sentence, so the first sentence is always the same, and always very blunt.

> Edit is not in the current edit scope. Change scope to \<NameOfEditScope\> to disable.

Reply:
Switched over to the suggested `Edit is not in the current edit scope. Change scope to <NameOfEditScope> to disable.`

Also took the opportunity to conform the Inspector's existing "EditScope (EditScope)" messages to the same style in c4ddc45.